src/IO/Resources/AnimationsLoader.cs

```diff
@@ -1937,7 +1934,7 @@ internal class IndexAnimation
         private byte[] _uopReplaceGroupIndex;
         public bool IsUOP => (Flags & ANIMATION_FLAGS.AF_USE_UOP_ANIMATION) != 0;
 
-        public bool HasBodyConversion => (GraphicConversion & 0x8000) == 0 && BodyConvGroups != null;
```

Comment:
Does the code check for `BodyConvGroups != null` when call it?

Reply:
I added a check that should be equivalent to this on line 1044